vue/src/modules/Workspace/view/users/WorkspaceUsers.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div v-if="loading">
+    <content-spinner show/>
+  </div>
+  <div v-else class="workspace-users">
+    <b-card class="rounded-0 border-0" style="box-shadow: 2px 2px 4px lightgrey;">
+      <no-data-available v-if="!users" height="100"/>
+      <table class="table table-striped" v-else>
+        <thead class="title text-center">
+        <tr>
+          <th>Image</th>
+          <th>Name</th>
+          <th>Roles</th>
+        </tr>
+        </thead>
+        <tbody class="text-center">
+        <tr class="py-0" v-for="user in users" :key="user.id">
+          <td class="align-middle py-1" style="width: 20%">
+            <b-img width="34" class="user-image rounded-circle mx-5"
+                   :src="user.image_url ? user.image_url : '/assets/img/avatar.svg'"></b-img>
+          </td>
+          <td class="align-middle py-1 text-dark font-weight-bold">
+            {{ user['display_name'] }}
+          </td>
+          <td class="align-middle py-2" style="width: 30%">
+            <span v-for="(role,index) in user.roles" :style="{'color': roleColor[role] || roleColor['default']}"
+                  class='px-1' :key="index">#{{ role }}</span></td>
```

Comment:
@ll-bat It is not best practice to have index as key in for loop, key should be unique. Try something like this: ``${role}-${index}``